src/docker.ts

```diff
@@ -71,6 +74,14 @@ async function getSHATag(repository: string): Promise<string> {
   return `${repository}:${res.stdout.trim()}`;
 }
 
+export async function version(): Promise<void> {
```

Comment:
This is not really used, but it's useful for getting docker compatibility information, so i will keep it for future debugging easiness.